pkg/api/resolver_mutation_performer.go

```diff
@@ -153,6 +172,12 @@ func (r *mutationResolver) PerformerUpdate(ctx context.Context, input models.Per
 		updatedPerformer.Checksum = &checksum
 	}
 
+	if *input.Deathdate != "" {
+		if utils.IfDateHigherThanAnotherDate(*input.Birthdate, *input.Deathdate) {
```

Comment:
A `utils` function is not necessary. Create a local function (something like `validateDeathDate`) that converts the strings into dates and basically does what the `utils` function is doing.